lib/decidim/calendar/version.rb

```diff
@@ -3,10 +3,10 @@
 module Decidim
   # Holds decidim-calendar version
   module Calendar
-    DECIDIM_VERSION = [">= 0.25.0", "< 0.27"].freeze
+    DECIDIM_VERSION = [">= 0.27.0", "< 0.28"].freeze
 
     def self.version
-      "0.26.0"
+      "0.27.4"
```

Comment:
```suggestion
      "0.27.0"
```
This is the version of the plugin, no minor yet!